.github/workflows/labeler/set-labels.js

```diff
@@ -0,0 +1,63 @@
+const fs = require('fs');
+const { execSync } = require('child_process');
+
+module.exports = async ({ github, context }) => {
+    const { repo : { owner, repo }} = context;
+
+    const allArtifacts = await github.rest.actions.listWorkflowRunArtifacts({
```

Comment:
Nice! I was unsure how we would manage the download artifacts for the correct workflow! 🎉

Reply:
It's nice that the `workflow_run` has the implicit id of the original workflow.